### ISSUE TITLE
fstcompile doesn't show help message

### ISSUE THREAD

**Issue by yzmyyff** [wenet-e2e/wenet#512]
**Describe the bug**
fstcompile doesn't show help message.

**To Reproduce**
```shell
cd runtime/server/x86
mkdir build && cd build && cmake .. && cmake --build .
cd ../fc_base/openfst-subbuild/openfst-populate-prefix/bin
./fstcompile --help
```
Error log:
```
WARNING: Logging before InitGoogleLogging() is written to STDERR
F0715 15:19:40.602166 45002 flags.cc:101] SetFlags: Bad option: --help
*** Check failure stack trace: ***
[1]    45002 abort (core dumped)  ./fstcompile --help
```

**Comment by pengzhendong** [comment]
I am fixing this bug. You could install the openfst directly (without the patch we provided), and use the binary.

**Comment by pengzhendong** [comment]
fixed #529
